packages/relay/src/lib/eth.ts

```diff
@@ -1068,7 +1068,7 @@ export class EthImpl implements Eth {
   }
 
   static nullableNumberTo0x(input: number | BigNumber): string | null {
-    return input === null ? null : EthImpl.numberTo0x(input);
+    return input === null || input === undefined ? null : EthImpl.numberTo0x(input);
```

Comment:
The condition can be combined into `input == null`, this way it checks for both null and undefined

Reply:
AH good call, let me make this change and push

Reply:
Updated